code/modules/organs/internal/lungs/lung_gasses.dm

```diff
@@ -128,10 +128,10 @@
 		if(!H.co2overloadtime) // If it's the first breath with too much CO2 in it, lets start a counter, then have them pass out after 12s or so.
 			H.co2overloadtime = world.time
 		else if(world.time - H.co2overloadtime > 120)
-			H.Paralyse(3)
-			H.adjustOxyLoss(3) // Lets hurt em a little, let them know we mean business
-			if(world.time - H.co2overloadtime > 300) // They've been in here 30s now, lets start to kill them for their own good!
-				H.adjustOxyLoss(8)
+			H.adjustOxyLoss(1) // Lets hurt em a little, let them know we mean business
+			if(world.time - H.co2overloadtime > 600) // They've been in here 60s now, lets start to kill them for their own good!
+				H.adjustOxyLoss(3)
+				H.Paralyse(3)
```

Comment:
Yeah not the damage reduction, the paralyze being gated behind 600 seconds instead of the 12. 

It's meant to be a gas that fucks you up before you can realize it. Otherwise its really obvious you are suffering from co2 poisoning before this ever occurs.